hax/hax/handler.py

```diff
@@ -113,6 +117,29 @@ def _update_process_status(self, p: WorkPlanner, motr: Motr,
                 broadcast_hax_only=broadcast_hax_only)
         self.consul.update_process_status(event)
 
+        # If we are receiving M0_CONF_HA_PROCESS_STARTED for M0D processes
+        # then we will check if all the M0D processes on the local node are
+        # started. If yes then we are going to send node online event to
+        # MessageBus
+        # if ((event.chp_type ==
+        #      m0HaProcessType.M0_CONF_HA_PROCESS_M0D) and
+        #    (event.chp_event == m0HaProcessEvent.M0_CONF_HA_PROCESS_STARTED)):
+        if event.chp_event == m0HaProcessEvent.M0_CONF_HA_PROCESS_STARTED:
+            util: ConsulUtil = ConsulUtil()
```

Comment:
I think all this should be moved to Ha module in Ha class. Ha.post() or Ha.send must be invoked with object type, id and status. This ID should be a Hare ID and the corresponding global id must be retrieved from Consul in HA module.

Reply:
Done